### ISSUE TITLE
test failing: Failed to load module 'ASCII::SingleLine'

### ISSUE THREAD

**Issue by perlpunk** [perlancar/perl-Data-Format-Pretty-Console#2]
Hi,

with version 0.391 we are getting this error message during tests:

https://build.opensuse.org/package/live_build_log/devel:languages:perl:autoupdate/perl-Data-Format-Pretty-Console/standard/x86_64
```
[   19s] load_module_with_optional_args(): Failed to load module 'ASCII::SingleLine' (all prefixes tried: BorderStyle) at /usr/lib/perl5/vendor_perl/5.32.1/Module/Load/Util.pm line 71.
[   19s] # Tests were run but no plan was declared and done_testing() was not seen.
[   19s] # Looks like your test exited with 25 just after 20.
[   19s] t/01-basic.t ............. 
[   19s] Dubious, test returned 25 (wstat 6400, 0x1900)
```
I already reported this in #1 but should have created an extra issue since it's a different error.

**Comment by perlancar** [comment]
Could you confirm that updating Text::ANSITable to 0.603 or 0.604 fixes this?

**Comment by perlpunk** [comment]
Thanks. Text::ANSITable has a new dependency alias::module which we don't have in openSUSE yet. I have to add that first. This process can take some days.

**Comment by perlpunk** [comment]
Yes, it's fixed with Text::ANSITable 0.604
https://build.opensuse.org/package/show/devel:languages:perl:autoupdate/perl-Data-Format-Pretty-Console
